core/mutator.ts

```diff
@@ -236,65 +236,63 @@ export class Mutator extends Icon {
   override updateEditable() {
     super.updateEditable();
     if (!this.getBlock().isInFlyout) {
-      if (this.getBlock().isEditable()) {
-        if (this.iconGroup_) {
-          dom.removeClass(this.iconGroup_, 'blocklyIconGroupReadonly');
-        }
-      } else {
+      if (!this.getBlock().isEditable) {
         // Close any mutator bubble.  Icon is not clickable.
         this.setVisible(false);
-        if (this.iconGroup_) {
-          dom.addClass(this.iconGroup_, 'blocklyIconGroupReadonly');
-        }
+      }
+      if (this.iconGroup_) {
+        dom.removeClass(this.iconGroup_, 'blocklyIconGroupReadonly');
```

Comment:
seems like we are now always removing the class, but before we were adding it or removing it depending on the new editable state. I don't see where we're ever adding the class, is this change intended?

Reply:
And this is why we have code reviews! Nope, I misread and thought they were doing the same thing. Good catch.